conduit/pipeline.go

```diff
@@ -296,42 +310,52 @@ func (p *pipelineImpl) addMetrics(block data.BlockData, importTime time.Duration
 // Start pushes block data through the pipeline
 func (p *pipelineImpl) Start() {
 	p.wg.Add(1)
+	retry := 0
 	go func() {
 		defer p.wg.Done()
 		// We need to add a separate recover function here since it launches its own go-routine
 		defer HandlePanic(p.logger)
 		for {
 		pipelineRun:
+			metrics.PipelineRetryGauge.Set(float64(retry))
```

Comment:
Because of the way Prometheus collects metrics, I think this needs to be a counter. Since the metrics server scrapes metrics at a configurable interval, we could easily miss retry attempts unless we measure retries across all rounds.

Reply:
A histogram/summary might also work

Reply:
updated